Code/GraphMol/MolHash/HashToString.cpp

```diff
@@ -8,12 +8,12 @@
 //  which is included in the file license.txt, found at the root
 //  of the RDKit source tree.
 //
+#include <algorithm>
```

Comment:
In case you're wondering, this is moved up because of a bug in boost 1.55 (https://svn.boost.org/trac/boost/ticket/9421) - we're stuck on 1.55 for the time being - so it's much easier for us to just move this up a tiny bit.